WordPress/src/main/java/org/wordpress/android/ui/main/WPMainActivity.java

```diff
@@ -251,7 +251,7 @@ public void onPageScrolled(int position, float positionOffset, int positionOffse
                     }
                 }
             } else {
-                if (hasMagicLinkLoginIntent()) {
+                if (hasMagicLinkLoginIntent() || hasMagicLinkSignupIntent()) {
```

Comment:
Is the `hasMagicLinkSignupIntent()` condition needed? Can it be true while `hasMagicLinkLoginIntent()` is `false`?

Reply:
That is true that `hasMagicLinkLoginIntent()` must be `true` if `hasMagicLinkSignupIntent()` is `true` since they have the same scheme and host.  I removed `hasMagicLinkSignupIntent()` from this conditional in 722916d19184d796e490cff2bc36bb604e1c1bd2.